lib/Elastica/QueryBuilder/DSL/Query.php

```diff
@@ -196,7 +195,7 @@ public function field()
      */
     public function filtered(AbstractQuery $query = null, AbstractFilter $filter = null)
     {
-        return new Filtered($query, $filter);
+        throw new DeprecatedException('Use BoolQuery instead. Filtered query is deprecated since ES 2.0.0-beta1 and this class will be removed in further Elastica releases.');
```

Comment:
And if Filtered deprecated and can't be used now, than constructor must throw DeprecatedException. Main idea to prevent developer do something that make no sense now.